eng/pipelines/templates/steps/vmr-prepare.yml

```diff
@@ -10,9 +10,12 @@ steps:
     displayName: Clone dotnet/dotnet
     path: vmr
     clean: true
+    fetchTags: true
+    fetchDepth: 0
 
-  - script: |
-      git checkout --track origin/${{ parameters.vmrBranch }}
-      echo "##vso[task.setvariable variable=vmrBranch]${{ parameters.vmrBranch }}"
+  - powershell: |
+      $branchName = "${{ parameters.vmrBranch }}" -replace "refs/heads/", ""
+      git checkout --track origin/$branchName || exit 1
```

Comment:
This change seems to be the reason for the Windows VMR failures.

Reply:
```suggestion
      $branchName = "${{ parameters.vmrBranch }}" -replace "refs/heads/", ""
      if (-not (git checkout --track origin/$branchName)) {
          throw "Failed to checkout branch $branchName"
      }
```